test/e2e/incluster/leaderelection_test.go

```diff
@@ -30,7 +30,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-func testLeaderElection(namespace string) {
```

Comment:
Should `namespace` be getting used below instead of `f.Deployment.Namespace.Name`? Seems like the latter would just be `projectcontour` which is not where this test is supposed to be getting run

Reply:
I think this test is just checking leases/events are created as expected on leader changes and these resources should just exist in the namespace the contour is running in so there isn't actually any need to namespace-scope this one

Reply:
Ah yeah OK nvm